Feature/rework cluster upgrade (#7180)
inverse-inc/packetfence: docs/cluster/appendix.asciidoc

/usr/local/pf/bin/pfcmd service galera-autofix stop
----

==== Detaching and upgrading node C

In order to be able to work on node C, we first need to stop all the
PacketFence application services on it:

[source,bash]
----
/usr/local/pf/bin/pfcmd service pf stop
----

IMPORTANT: `packetfence-config` should stay started in order to run `/usr/local/pf/bin/cluster/node` commands.
  

> Next, you can upgrade your operating system and/or PacketFence on node C.

Reviewer: ```suggestion
In the next following steps, you will be upgrading PacketFence on node C.
```
